ios/RNSScreenStackHeaderConfig.m

```diff
@@ -138,10 +138,23 @@ - (void)updateViewControllerIfNeeded
   }
 }
 
+- (void)layoutNavigationControllerView
+{
+  UIViewController *vc = _screenView.controller;
+  UINavigationController *navctr = (UINavigationController *)vc.parentViewController;
```

Comment:
Maybe we could just call `vc.navigationController`? If it gives the proper controller, we wouldn't need to cast it.